extension/linkify-urls-in-code.js

```diff
@@ -28,7 +28,7 @@ window.linkifyURLsInCode = (() => {
 		.toArray()
 		.forEach(blob => {
 			if (hasURL(blob.innerHTML)) {
```

Comment:
True that, I guess the initial author wanted to avoid running `match()` in general in case there is no link present. Also, this would mean we’d have to clean up the tests since this line is the only place where we’re using `hasURL` function at all.